libraries/chain/include/eosio/chain/account_object.hpp

```diff
@@ -29,7 +29,6 @@ namespace eosio { namespace chain {
       shared_vector<char>  abi;
 
       void set_abi( const eosio::chain::contracts::abi_def& a ) {
-         // Added resize(0) here to avoid bug in boost vector container
```

Comment:
I should restore this comment... not sure why I took it out

Reply:
I mean.. am I the only one who is seriously bothered by this? @PaulCalabrese did we ever ask boost folks about this?

Reply:
you are not the only one... but I can confirm that taking it out -OR- moving it all into the `create` call causes tests to fail in that same odd bug.   I had tried to at least consolidate the abi and the code "hack" into the same logical place but it seems there is some corruption that can occur between `create`/constructor and future calls.